Pickers/SWActionSheet.m

```diff
@@ -260,7 +260,8 @@ - (BOOL)prefersStatusBarHidden
 
 - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldReceiveTouch:(UITouch *)touch
 {
-    if (touch.view.tag == sheetBackgroundTag) {
```

Comment:
@palmtrae please remove the tag and sheetBackgroundTag code too in your PR.

Reply:
Done :)